2021-05-29 17:45:03,233 INFO    MainThread:29823 [wandb_setup.py:_flush():69] setting env: {}
2021-05-29 17:45:03,233 INFO    MainThread:29823 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-29 17:45:03,233 INFO    MainThread:29823 [wandb_init.py:_log_setup():336] Logging user logs to /home/mohan/research/masters/robotics/robolfd/examples/offline/wandb/run-20210529_174503-257rdv46/logs/debug.log
2021-05-29 17:45:03,233 INFO    MainThread:29823 [wandb_init.py:_log_setup():337] Logging internal logs to /home/mohan/research/masters/robotics/robolfd/examples/offline/wandb/run-20210529_174503-257rdv46/logs/debug-internal.log
2021-05-29 17:45:03,233 INFO    MainThread:29823 [wandb_init.py:init():369] calling init triggers
2021-05-29 17:45:03,233 INFO    MainThread:29823 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2021-05-29 17:45:03,233 INFO    MainThread:29823 [wandb_init.py:init():418] starting backend
2021-05-29 17:45:03,233 INFO    MainThread:29823 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-29 17:45:03,240 INFO    MainThread:29823 [backend.py:ensure_launched():132] starting backend process...
2021-05-29 17:45:03,244 INFO    MainThread:29823 [backend.py:ensure_launched():136] started backend process with pid: 29838
2021-05-29 17:45:03,244 INFO    MainThread:29823 [wandb_init.py:init():423] backend started and connected
2021-05-29 17:45:03,245 INFO    MainThread:29823 [wandb_init.py:init():465] updated telemetry
2021-05-29 17:45:03,245 INFO    MainThread:29823 [wandb_init.py:init():484] communicating current version
2021-05-29 17:45:03,639 INFO    MainThread:29823 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.31 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-05-29 17:45:03,639 INFO    MainThread:29823 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-29 17:45:03,967 INFO    MainThread:29823 [wandb_init.py:init():522] starting run threads in backend
2021-05-29 17:45:04,878 INFO    MainThread:29823 [wandb_run.py:_console_start():1565] atexit reg
2021-05-29 17:45:04,878 INFO    MainThread:29823 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-05-29 17:45:04,878 INFO    MainThread:29823 [wandb_run.py:_redirect():1417] Redirecting console.
2021-05-29 17:45:04,878 INFO    MainThread:29823 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-29 17:45:04,878 INFO    MainThread:29823 [wandb_init.py:init():546] run started, returning control to user process
2021-05-29 17:47:35,366 INFO    MainThread:29823 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 1
2021-05-29 17:47:35,367 INFO    MainThread:29823 [wandb_run.py:_restore():1507] restore
2021-05-29 17:47:37,372 INFO    MainThread:29823 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 792
  total_bytes: 792
}

2021-05-29 17:47:37,761 INFO    MainThread:29823 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 792
  total_bytes: 792
}

2021-05-29 17:47:37,993 INFO    MainThread:29823 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 792
  total_bytes: 1182
}

2021-05-29 17:47:38,638 INFO    MainThread:29823 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8307
  total_bytes: 8307
}

2021-05-29 17:47:38,740 INFO    MainThread:29823 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8307
  total_bytes: 8307
}

2021-05-29 17:47:38,841 INFO    MainThread:29823 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8307
  total_bytes: 8307
}

2021-05-29 17:47:38,942 INFO    MainThread:29823 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8307
  total_bytes: 8307
}

2021-05-29 17:47:39,044 INFO    MainThread:29823 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8307
  total_bytes: 8307
}

2021-05-29 17:47:39,145 INFO    MainThread:29823 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8307
  total_bytes: 8307
}

2021-05-29 17:47:40,187 INFO    MainThread:29823 [wandb_run.py:_show_files():1879] logging synced files
